webapp/src/main/resources/templates/email/todays-releases-email.html

```diff
@@ -1,66 +1,46 @@
-            margin-bottom: 0.5rem;
-        }
-        .cover {
-            border-radius: 5px;
-            width: 9rem;
-            height: 9rem;
-            object-fit: cover;
-            border: 1px solid #464b53;
-        }
-        .release-box {
-            float: left;
-            padding-left: 4%;
-            width: 66%;
-            margin-bottom: 0.5rem;
-            box-sizing: border-box;
-        }
-        @media screen and (max-width: 650px) {
-            .cover-box {
-                width: 100%;
-            }
-            .cover {
-                width: 12rem;
-                height: 12rem;
-                display: block;
-                margin-left: auto;
-                margin-right: auto;
-            }
-            .release-box {
-                width: 100%;
-                text-align: center;
-                margin-bottom: 1.5rem;
-            }
-        }
-    </style>
+    <meta http-equiv="Content-Type" content="text/html charset=UTF-8" />
 </head>
-<body>
-<div layout:fragment="content">
-    <p class="text-l">Here are your favorite band's releases coming out today:</p>
-    <div th:each="release: ${todaysReleases}">
-        <div class="cover-box">
-            <img class="cover" src="#" th:src="${release.coverUrl}" alt="Release Cover" width="150"/>
-        </div>
-        <div class="release-box">
-            <div class="release-title">
-                <span class="text-xl text-bold" th:text="${release.artist}"></span><br />
-                <span class="text-l text-bold" th:text="${release.albumTitle}"></span><br />
-                <span class="text-bold" th:text="${release.releaseDateAsDisplayString}"></span>
-            </div>
-        </div>
-        <div style="clear:both;"></div>
-    </div>
-</div>
+<body style="background-color:#02040A;padding:50px 0;color:#a4aab0;font-family:Roboto, -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Roboto, &quot;Helvetica Neue&quot;, Arial, sans-serif">
+<table align="center" role="presentation" cellSpacing="0" cellPadding="0" border="0" width="100%" style="max-width:37.5em;background-color:#0C0E14;border:1px solid #21262d;padding:45px;border-radius:5px">
+    <tr style="width:100%">
+        <td><img alt="Metal Detector Logo" src="https://s3.eu-central-1.amazonaws.com/rocks.metal-detector.s3.images/images/assets/logo.png" width="450" style="display:block;outline:none;border:none;text-decoration:none;margin-left:auto;margin-right:auto" />
+            <table align="center" border="0" cellPadding="0" cellSpacing="0" role="presentation" width="100%">
+                <tbody>
+                <tr>
+                    <td>
+                        <h1 style="font-size:30px;font-weight:bold;text-align:center">Hi <span th:text="${username}">username</span>,</h1>
+                        <p style="font-size:18px;line-height:28px;margin:16px 0;font-weight:300;margin-bottom:25px">Here are your favorite band's releases coming out today:</p>
```

Comment:
```suggestion
                        <p style="font-size:18px;line-height:28px;margin:16px 0;font-weight:300;margin-bottom:25px">Here are your favorite bands' releases coming out today:</p>
```
Checked this one with Paula 😉